csharp/IceStorm/clock/subscriber/Subscriber.cs

```diff
@@ -2,6 +2,7 @@
 
 using Demo;
 using System;
+using System.Threading;
 using System.Collections.Generic;
 using System.Configuration;
 using System.Diagnostics;
```

Comment:
Sort using directives 
```suggestion
using System.Collections.Generic;
using System.Configuration;
using System.Diagnostics;
using System.Threading;
```